caravel/assets/javascripts/SqlLab/components/QueryTable.jsx

```diff
@@ -27,7 +27,7 @@ const defaultProps = {
 };
 
 
-class QueryTable extends React.Component {
+class QueryTable extends React.PureComponent {
```

Comment:
`PureComponent` implements its own `shouldComponentUpdate` that checks for shallow difference in `this.state.*` and `this.props.*` against the next values.